xclim/core/indicator.py

```diff
@@ -369,8 +375,8 @@ def update_attrs(self, ba, das):
             attrs["cell_methods"] += " " + out.pop("cell_methods")
 
         attrs["history"] = update_history(
-            f"{self.identifier}{ba.signature.replace(parameters=cp.values())}",
-            new_name=vname,
+            f"{identifier or cls.identifier}{ba.signature.replace(parameters=cp.values())}",
```

Comment:
Not sure I understand why identifier would be different from `cls.identifier`.

Reply:
In MultiIndicator the "identifier" passed to `update_attrs` would be something like : "FWI.DC", whereas the class identifier is "FWI".